modules/cms-kit/src/Volo.CmsKit.Admin.Application.Contracts/Volo/CmsKit/Admin/Blogs/IBlogFeatureAdminAppService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Volo.CmsKit.Admin.Blogs
+{
+    public interface IBlogFeatureAdminAppService
+    {
+        Task SetAsync(Guid blogId, BlogFeatureDto dto);
```

Comment:
We can change the input dto name as `BlogFeatureInputDto`

Reply:
I'm usşng same dto as return type. They have completely same properties. Are you suggesting to create seperated 2 dtos? @cotur

Reply:
Yes, please use two different dto.